### ISSUE TITLE
Conflict with Overmind Tmux integration

### ISSUE THREAD

**Issue by tomdracz** [withfig/fig#1186]
### Sanity checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have typed 'git ' and included a screenshot of the Debugger (click the Fig icon in the macOS menu bar)

### Description:
Using iTerm 2 + Fish shell & OMF. Fig seems to break something related to tmux.

We're using https://github.com/DarthSim/overmind to manage multiple processes in tmux sessions. But when trying to start it with Fig installed, it seems like the processes do not end up running. Screenshot below:
![Screenshot 2022-03-25 at 10 22 39](https://user-images.githubusercontent.com/6048970/160105048-fc2658a8-5bd2-484b-a60e-1710fc9ca601.png)

Vs running without Fig:
![Screenshot 2022-03-25 at 10 38 28](https://user-images.githubusercontent.com/6048970/160105447-5c593ded-2880-4f3c-85b5-623149b302f3.png)

Do note that this breaks with Fig even installed. Disabling all integrations and/or quitting Fig makes this problem persists. Only after uninstalling I was able to get this up and running again

### Debugger screenshot:
<img width="533" alt="Screenshot 2022-03-25 at 10 30 00" src="https://user-images.githubusercontent.com/6048970/160105535-ea6aeb8f-38ce-4367-b21a-74ccef31e604.png">

### Environment

```bash
# Fig Diagnostics
## Fig details:
  - Fig version: Version 1.0.56 (B416) [British]
  - Bundle path: /Applications/Fig.app
  - Autocomplete: true
  - Settings.json: true
  - Accessibility: true
  - Number of specs: 0
  - Symlinked dotfiles: false
  - Only insert on tab: false
  - Keybindings path:
  - Installation Script: true
  - PseudoTerminal Path: /opt/homebrew/bin/:/Users/tomdracz/.local/bin:/Users/tomdracz/.fig/bin:/Users/tomdracz/.cargo/bin:/Users/tomdracz/.rbenv/shims:/Users/tomdracz/.nodenv/shims:/Library/Apple/usr/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin
  - SecureKeyboardInput: false
  - SecureKeyboardProcess: <none>
## Hardware Info:
  - Model Name: MacBook Pro
  - Model Identifier: MacBookPro17,1
  - Chip: Apple M1
  - Cores: 8 (4 performance and 4 efficiency)
  - Memory: 16 GB
## OS Info:
  - macOS 12.2.1 (21D62)
## Environment:
  - User Shell: /opt/homebrew/bin/fish
  - Current Directory: /Users/tomdracz/projects/booking-system
  - CLI Installed: true
  - Executable Location: /Users/tomdracz/.local/bin/fig
  - Current Window ID: 31141/% (com.googlecode.iterm2)
  - Active Process: fish (20165) - /dev/ttys005
  - Environment Variables:
    - FIG_TERM=1
    - TERM=xterm-256color
    - FIG_INTEGRATION_VERSION=8
    - PATH=/opt/homebrew/bin/:/Users/tomdracz/.local/bin:/Users/tomdracz/.fig/bin:/Users/tomdracz/.cargo/bin:/Users/tomdracz/.rbenv/shims:/Users/tomdracz/.nodenv/shims:/Library/Apple/usr/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin
    - FIG_TERM_VERSION=4.3.0
    - TERM_SESSION_ID=w0t0p0:DB1F1881-289B-4E70-B68E-A1095E887FFA
## Integrations:
  - SSH: false
  - TMUX: false
  - iTerm: installed!
  - Hyper: application is not present.
  - Visual Studio Code: installed!
  - Docker: true
```

**Comment by tomdracz** [comment]
This seems to be fixed when running the latest version. Thanks!

cc @SeparateRecords
